src/tools/utils/sequences.ts

```diff
@@ -133,3 +125,51 @@ export const removeFeaturesWithUnknownModifier = (features?: FeatureData) =>
       start.modifier !== LocationModifier.UNKNOWN &&
       end.modifier !== LocationModifier.UNKNOWN
   );
+
+export const createGappedFeature = (
+  feature: ProcessedFeature,
+  sequence: string
+) => {
+  /*
+  input: feature and sequence are both 1-based
+  */
+  const BLOCK = /(?<insertion>[-]+)|(?<protein>[^-]+)/g;
```

Comment:
I remembered we had issues before with named capture groups and Firefox support. I would say that now since my latest changes in the Babel config this is transpiled/polyfilled correctly and that shouldn't be a problem. But I would recommend double-checking on Firefox 77 or less (FF78 is the first supporting this)

Reply:
Checked on FF72 and all good.